include/crypto/modes.h

```diff
@@ -144,6 +144,12 @@ struct gcm128_context {
 #endif
 };
 
+/* GHASH functions */
+void gcm_init_4bit(u128 Htable[16], const u64 H[2]);
+void gcm_ghash_4bit(u64 Xi[2], const u128 Htable[16],
+                    const u8 *inp, size_t len);
+void gcm_gmult_4bit(u64 Xi[2], const u128 Htable[16]);
+
```

Comment:
These functions need to be renamed with `ossl_` prefix.

Reply:
These functions are all defined within the crypto/mode/gcm128.c file; and would require modifying A LOT of code in there. I'm just using these functions from that file. I think it's out of scope to rename these.

Reply:
Ah, OK, I did not realize these functions are static only in no-asm build.  OK to keep them without renaming then.

Reply:
I decided to create an `ossl_gcm_XXX_4bit()` wrapper around those static and ASM APIs; this at least gives the desired "public" renaming without mucking up the assembly. Still waiting for all the checks to complete.